tests/test_rp_vars.py

```diff
@@ -47,7 +44,7 @@ def test_get_all_rp_vars(new_solver_session_no_transcript) -> None:
     assert len(all_vars) == pytest.approx(9000, 20)
 
     # CaseFile comparison, note that the PyFluent work dir is not necessarily the same as the Fluent work dir
-    case = CaseReader(case_filepath=str(Path(pyfluent.EXAMPLES_PATH) / case_path))
+    case = CaseReader(case_filepath=path(case_path))
```

Comment:
This `path(case_path)` is an example where it would break when running this test with standalone Fluent without the changes I suggested, as `case_path` is already going to be an absolute path to begin with

Reply:
We get same output with both file name and file path.

![image](https://github.com/ansys/pyfluent/assets/106588300/cd611f58-de83-49c2-b471-1e391c15c85c)